### PR TITLE
Document -Xtgc:allocation

### LINKED ISSUE #1296: Document -Xtgc:allocation
**Issue or pull request number:**
https://github.com/eclipse-openj9/openj9/pull/19070

**Overview:**
-Xtgc:allocation has existed 'forever' but just recently (for 0.46) has been enhanced so it may be more useful for users to find thread specific allocation stats, hence more reasons to document it.

An attempt here: https://github.com/eclipse-openj9/openj9-docs/pull/1295

**Release target:**
Eclipse OpenJ9 0.46

**Applies to the following JDK versions:**
All

**Applies to the following platforms:**
All

### PR DESCRIPTION
**Pull request by Sreekala-Gopakumar**
https://github.com/eclipse-openj9/openj9-docs/issues/1296

Added information about the allocation parameter. Updated the What's new topic.

Closes #1296
Signed-off-by: Sreekala Gopakumar sreekala.gopakumar@ibm.com

### CODE REVIEW

**Comment by Sreekala-Gopakumar** [comment]
Jenkins doc stage

**Comment by Sreekala-Gopakumar** [comment]
@amicic, @pshipton - I have updated the document. Please check and confirm. Thanks!

**Comment by Sreekala-Gopakumar** [comment]
Jenkins doc stage

**Comment by Sreekala-Gopakumar** [comment]
@amicic, @pshipton - I have updated the document. Please check and confirm. Thanks!

**Comment by Sreekala-Gopakumar** [comment]
Jenkins doc stage

**Comment by Sreekala-Gopakumar** [comment]
@amicic - I have updated the document. Please check and confirm that the changes are ok. Thanks!

**Comment by Sreekala-Gopakumar** [comment]
Jenkins doc stage

**Comment by amicic** [comment]
I don't see a need for 'enhanced', since this is first time introduced in the docs and most don't know about it. Simple 'option prints' is sufficient to me.

**Review by amicic — APPROVED** [review]

File: docs/version0.46.md [hunk 1]
```diff
@@ -72,6 +73,10 @@ The extended HCR feature is deprecated in this release and will be removed in a
 
 When the `jcmd` tool sends a command to a running VM, the command might throw the `Socket Exception` error on Windows platform. Instead of failing the attaching request, you can specify the number of times the tool retries attaching to the target VM with the new system property, [`-Dcom.ibm.tools.attach.retry`](dcomibmtoolsattachretry.md).
 
+### `-Xtgc:allocation` enhanced to report core allocation cache stats per thread
+
+The [`-Xtgc:allocation`](xtgc.md#allocation) option is enhanced to print thread specific allocation cache (TLH) stats in addition to the cumulative allocation stats.
```
Comment (keithc-ca) [inline docs/version0.46.md:78]: Suggest a hyphen in "thread-specific".

**Comment by Sreekala-Gopakumar** [comment]
Jenkins doc stage

**Comment by Sreekala-Gopakumar** [comment]
@amicic @keithc-ca @pshipton - I have made the changes. Please check and confirm. If everything is ok, please merge. Thanks!

File: docs/version0.46.md [hunk 2]
```diff
@@ -72,6 +73,10 @@ The extended HCR feature is deprecated in this release and will be removed in a
 
 When the `jcmd` tool sends a command to a running VM, the command might throw the `Socket Exception` error on Windows platform. Instead of failing the attaching request, you can specify the number of times the tool retries attaching to the target VM with the new system property, [`-Dcom.ibm.tools.attach.retry`](dcomibmtoolsattachretry.md).
 
+### `-Xtgc:allocation` report includes core allocation cache stats per thread
+
+The [`-Xtgc:allocation`](xtgc.md#allocation) option prints thread-specific allocation cache (TLH) stats in addition to the cumulative allocation stats.
```
Comment (keithc-ca) [inline docs/version0.46.md:78]: Wouldn't it be better not to abbreviate "statistics"?

**Comment by Sreekala-Gopakumar** [comment]
Jenkins doc stage

File: docs/version0.46.md (hunk 2 above)
Comment (Sreekala-Gopakumar) [reply docs/version0.46.md]: Done. Thanks!

Comment (Sreekala-Gopakumar) [reply docs/version0.46.md]: Any other changes? Can this be merged?

**Review by keithc-ca — APPROVED** [review]

**Pull request merged**